src/graphic/Text.js

```diff
@@ -67,6 +67,7 @@ Text.prototype = {
                 style.textAlign,
                 style.textVerticalAlign,
                 style.textPadding,
+                textContain.getLineHeight(style.font),
```

Comment:
How does line height to do with opacity? If they're not related, you should probably use another PR for it.